components/steering_wheel/include/IO/IO_Digital.h

```diff
@@ -0,0 +1,58 @@
+/**
+ * IO_Digital.h
+ * Digital IO Module Header
+ * @note To transfer this module to another system, the digitalInput_E and the
+ *      typedef'd IO_Digital_Flag_t must be modified to be congruent with the new system
+ */
+
+#pragma once
+
+/******************************************************************************
+ *                             I N C L U D E S
+ ******************************************************************************/
+
+#include "Types.h"
+#include "stm32f103xb.h" /**< Needed for GPIO_TypeDef */
```

Comment:
should include `gpio.h` instead. It'll do the same thing at the end of the day, but allows for further abstraction for other chips in the future.